test/import/test_wmd_importer.cpp

```diff
@@ -270,3 +271,107 @@ INSTANTIATE_TEST_SUITE_P(
+    graph.setData(vert, dumbVertex);
+    vertexData = graph.getData(vert);
+    EXPECT_EQ(vertexData.id, numVertices);
+
+    // Iterate over edges
+    EXPECT_NE(goldenTable.find(srcTok), goldenTable.end())
+        << "Failed to find edges with tok_id:" << srcTok << "\t with index: " << (vid - 1);
+    std::vector<std::uint64_t> goldenEdges = goldenTable[srcTok];
+
+    for (typename Graph::EdgeHandle eh : graph.edges(vert)) {
+      typename Graph::EdgeData eData = graph.getEdgeData(eh);
+
+      EXPECT_EQ(eData.src, srcTok);
+
+      typename Graph::VertexTokenID dstTok = graph.getTokenID(graph.getEdgeDst(eh));
+      EXPECT_EQ(eData.dst, dstTok);
+
+      auto mirrorTopology = graph.getTopologyID(dstTok);
+      auto masterTopology = graph.getGlobalTopologyID(dstTok);
+      if (mirrorTopology != masterTopology) {
+        // If global, and local have different value.
+        // It means current one have mirror. Mirror is local, but master is not.
+        ASSERT_TRUE(graph.isLocal(mirrorTopology));
+        ASSERT_TRUE(!graph.isLocal(masterTopology));
+        // Mirror must exist in mirror range.
+        auto it = graph.getMirrorRange();
+        ASSERT_TRUE(*it.begin() <= mirrorTopology && mirrorTopology < *it.end());
+      } else {
+        // If I don't have mirror, that could be because it is in local, or never be a destination
+        // from me.
+        if (graph.isLocal(masterTopology)) {
+          // If it is from me, it is in my master range.
+          auto it = graph.getMasterRange();
+          ASSERT_TRUE(*it.begin() <= masterTopology && masterTopology < *it.end());
+          // In mirror to master, this should never exist
+        }
+      }
+
+      auto goldenEdgeIt = std::find(goldenEdges.begin(), goldenEdges.end(), dstTok);
+      EXPECT_NE(goldenEdgeIt, goldenEdges.end())
+          << "Unable to find edge with src_tok: " << srcTok << "\tand dst_tok: " << dstTok
+          << "\tat vertex: " << (vid - 1);
+      ET dumbEdge = ET{numVertices, numVertices};
+      graph.setEdgeData(eh, dumbEdge);
+      eData = graph.getEdgeData(eh);
+      EXPECT_EQ(eData.src, numVertices);
+      EXPECT_EQ(eData.dst, numVertices);
+    }
+  }
+  graph.deinitialize();
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    SmallFiles, MirrorDLCSRInitEdgeList,
+    ::testing::Values(std::make_tuple("/pando/graphs/simple.el", 10),
+                      std::make_tuple("/pando/graphs/rmat_571919_seed1_scale10_nV1024_nE10447.el",
+                                      1024)));
+
+INSTANTIATE_TEST_SUITE_P(
+    DISABLED_BigFiles, MirrorDLCSRInitEdgeList,
+    ::testing::Values(
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale11_nV2048_nE22601.el", 2048),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale12_nV4096_nE48335.el", 4096),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale13_nV8192_nE102016.el", 8192),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale14_nV16384_nE213350.el", 16384)));
```

Comment:
Have you tested with these files?

Reply:
For 2048, yes, but not for the others.

Reply:
just deleted disabled and run it locally. If that works then we are good.

Reply:
UBSAN and Release passed.

DRVx takes too long...

Reply:
drvx we can ignore for now.